src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -422,11 +422,11 @@ public Vector(ReadOnlySpan<T> values)
             return result;
         }
 
-
         /// <summary>Multiplies a vector by the given scalar.</summary>
         /// <param name="factor">The scalar value.</param>
         /// <param name="value">The source vector.</param>
         /// <returns>The scaled vector.</returns>
+        [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why are we keeping `AggressiveInlining` for `factor * vector` but removing for `value * factor`? Just trying to understand the logic

Reply:
Because one forwards to `factor * vector` simply forwards to `vector * factor`. While the latter (`vector * factor`) contains the `for` loop for the software fallback and should not be implicitly inlined.
This matches what we do for the other operators on `Vector<T>`.